### PR TITLE
YQ: SDK with Object Storage Connection resource

### PR DESCRIPTION
**Pull request by lifthelm**
Added:
- YQ SDK
- Object Storage Connection resource

### CODE REVIEW

**Comment by dmchmk** [comment]
Hi, @uzhastik, can you please help me review this PR?

File: yandex/internal/yq/sdk/sdk.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	if conf.FolderID == "" {
+		return fmt.Errorf("\"folder_id\" is required to YQ SDK")
+	}
+
+	if conf.AuthToken == "" {
+		return fmt.Errorf("\"token\" is required to YQ SDK")
+	}
+
+	if conf.Endpoint == "" {
+		return fmt.Errorf("\"yq_endpoint\" is required to YQ SDK")
```
Comment (lifthelm) [inline yandex/internal/yq/sdk/sdk.go:32]: `to` -> `for`

Comment (dmchmk) [reply yandex/internal/yq/sdk/sdk.go:32]: > `to` -> `for`

@lifthelm Maybe you want to change it in code?

**Comment by dmchmk** [comment]
will be rewritten later

**Pull request closed without merging**